src/textual/screen.py

```diff
@@ -851,6 +853,26 @@ def _update_tooltip(self, widget: Widget) -> None:
             if tooltip.display and self._tooltip_widget is widget:
                 self._handle_tooltip_timer(widget)
 
+    def _maybe_clear_tooltip(self) -> None:
+        """Check if the widget under the tooltip still pertains to the tooltip.
+
+        If they differ, the tooltip will be removed.
```

Comment:
```suggestion
        """Check if the widget under the mouse cursor still pertains to the tooltip.

        If they differ, the tooltip will be removed.
```

Reply:
(Not 100% sure if this suggestion is correct)

Reply:
Kinda both really; but yes technically it's the mouse cursor (although the intent is that it's "under" the tooltip).